src/plugins/urbanDictionary/index.ts

```diff
@@ -18,14 +18,22 @@
 
 import { ApplicationCommandOptionType, sendBotMessage } from "@api/Commands";
 import { ApplicationCommandInputType } from "@api/Commands/types";
+import { Settings } from "@api/Settings";
 import { Devs } from "@utils/constants";
-import definePlugin from "@utils/types";
+import definePlugin, { OptionType } from "@utils/types";
 
 export default definePlugin({
     name: "UrbanDictionary",
     description: "Search for a word on Urban Dictionary via /urban slash command",
     authors: [Devs.jewdev],
     dependencies: ["CommandsAPI"],
+    options: {
+        resultsAmount: {
+            type: OptionType.NUMBER,
+            description: "The amount of results you want to get (+ Better results, - Slower)",
```

Comment:
The bit after the description doesn't really make sense. Are you trying to say pros/cons? If so, it'd be better to reword it into a sentence in brackets rather than shorthanding it:
```suggestion
            description: "The amount of results you want to get (more gives better results, but is slower)",
```